pkg/apis/componentconfig/cloudcore/v1alpha1/validation/validation.go

```diff
@@ -45,18 +43,6 @@ func ValidateCloudCoreConfiguration(c *v1alpha1.CloudCoreConfig) field.ErrorList
 	return allErrs
 }
 
-//ValidateLeaderElectionConfiguration validates part `l` and returns an errorList if it is invalid, the rest will be validated at run time
-func ValidateLeaderElectionConfiguration(l componentbaseconfig.LeaderElectionConfiguration) field.ErrorList {
```

Comment:
not pretty sure why this is removed, doesn't that mean `leaderelelction` is no longer supported?

Reply:
Now we will support active-active mode of cloudcore, so leaderelelction is no loanger needed. :)